test/organization/test.sh

```diff
@@ -393,6 +440,109 @@ function test_cdb_usertables_should_work_with_orgusers() {
     sql cdb_testmember_1 "DROP TABLE test_perms_priv"
 }
 
+function test_CDB_Group_Table_GrantRead_should_grant_select_and_RevokeAll_should_remove_it() {
```

Comment:
Yep, I'm adding even conf tests here to focus on getting things done first. Depending on timing and ugliness I'll refactor this before the final merge or plan a refactor afterwards. But existing "test framework" is working great for me :)